Week1/homework/app.js

```diff
@@ -1,11 +1,137 @@
+      title: "A study in sable",
+      language: "English",
+      author: "Mercedes Lackey",
+      published: 2016
+    },
+    'Heartless': {
+      title: "Heartless",
+      language: "English",
+      author: "Marissa Meyer",
+      published: 2016
+    },
+    'The_door_in_the_moon': {
+      title: "The door in the moon",
+      language: "English",
+      author: "Catherine Fisher",
+      published: 2015
+    },
+    'Harry_Potter_and_the_Methods_of_Rationality': {
+      title: "Harry Potter and the Methods of Rationality",
+      language: "English",
+      author: "Eliezer Yudkowsky",
+      published: 2016
+    },
+    'The_6th_extinction': {
+      title: "The 6th extinction",
+      language: "English",
+      author: "James Rollins",
+      published: 2015
+    },
+    'Rocket_science': {
+      title: "Rocket science",
+      language: "English",
+      author: "Deborah Lock",
+      published: 2015
+    },
+    'Eat_what_you_love_everyday': {
+      title: "Eat what you love everyday",
+      language: "English",
+      author: "Marlene Koch",
+      published: 2014
+    },
+    'Island_of_glass': {
+      title: "Island of glass",
+      language: "English",
+      author: "Nora Roberts",
+      published: 2016 
+    }
+  };
+
+
+
+
+
+  function createNewElement() {
+    let check = checkExistedList();
+    if (check === true) {
+      let newUl = document.createElement("ul");
+      newUl.setAttribute("id", "myBookList");
+      let newOl = document.createElement("ol");
+      newOl.setAttribute("id", "listTiltle");
+      let liTitle = document.createTextNode("Here is my book's list :");
+      newOl.appendChild(liTitle);
+      let parent = document.getElementById("listContainer");
+      newUl.appendChild(newOl);
+      parent.appendChild(newUl);
+
+      
+      for (let i = 0; i < bookTitles.length; i++) {
+        let newLi = document.createElement("li");
+        newLi.setAttribute("id", bookTitles[i]);
+        let bookName = document.createTextNode(libraryInformation[bookTitles[i]].title);
+        let bookAuthor = document.createTextNode('Writed by : ' + libraryInformation[bookTitles[i]].author);
+        let bookLanguage = document.createTextNode('Book Language : ' + libraryInformation[bookTitles[i]].language);
+        let publishedDate = document.createTextNode('Published on : ' + libraryInformation[bookTitles[i]].published);
+        let h2Tag = document.createElement("h2");
+        let h3Tag = document.createElement("h3");
+        let h4Tag = document.createElement("h4");
+        let h5Tag = document.createElement("h5");
+        let bookCover = document.createElement("img");
+
+        h2Tag.appendChild(bookName);
+        h3Tag.appendChild(bookAuthor);
+        h4Tag.appendChild(bookLanguage);
+        h5Tag.appendChild(publishedDate);
+        let imageSrc = 'images/' + bookTitles[i] + '.jpg';
+        bookCover.setAttribute('src', imageSrc);
+        bookCover.setAttribute('alt', bookName);
```

Comment:
`typeof bookName` is Object so that means the alternative text will be an object, which means you have this alt repeatedly `[object Text]`.
you can solve this by `bookName.textContent` or you may choose to change the content of the variable to be the existed `bookName`